src/main/java/com/epam/prd/jjop/helpers/arrays/Copy.java

```diff
@@ -0,0 +1,173 @@
+         * @throws NullPointerException if original is null.
+         * @throws ArrayStoreException - if an element copied from original is not of a runtime type that can be stored in an array of class newType.
+         * Arrays.copyOf methods for reference type array:
+         * <ul>
+         *    <li> {@link java.util.Arrays#copyOf(Object[], int)} - Copies the specified array, truncating or padding with null
+         *          characters (if necessary) so the copy has the specified length.
+         *    </li>
+         *    <li> {@link java.util.Arrays#copyOf(Object[], int, Class)} - Copies the specified array, truncating or padding with null
+         *         characters (if necessary) so the copy has the specified length.
+         *    </li>
+         *  </ul>
+         * Prints copies of different types arrays
+         * New array elements will be assigned from an existing value from an old array.
+         * If the length of the new array is greater than the old array in such cases all remaining elements
+         * will be assigned as zero or null.
+         */
+
+        boolean[] new_booleans = Arrays.copyOf(booleansArray, 6);
+        System.out.println("New Boolean Array: " + Arrays.toString(new_booleans));
+        byte[] new_bytes = Arrays.copyOf(bytesArray, 10);
+        System.out.println("New Byte Array: " + Arrays.toString(new_bytes));
+        char[] new_chars = Arrays.copyOf(charsArray, 7);
+        System.out.println("New Char Array: " + Arrays.toString(new_chars));
+        double[] new_doubles = Arrays.copyOf(doublesArray, 10);
+        System.out.println("New Double Array: " + Arrays.toString(new_doubles));
+        float[] new_floats = Arrays.copyOf(floatsArray, 10);
+        System.out.println("New Boolean Array: " + Arrays.toString(new_floats));
+        int[] new_ints = Arrays.copyOf(intsArray, 10);
+        System.out.println("New Int Array: " + Arrays.toString(new_ints));
+        long[] new_longs = Arrays.copyOf(longsArray, 12);
+        System.out.println("New Long Array: " + Arrays.toString(new_longs));
+        short[] new_shorts = Arrays.copyOf(shortsArray, 7);
+        System.out.println("New Short Array: " + Arrays.toString(new_shorts));
+        String[] new_strings = Arrays.copyOf(stringsArray, 5);
+        System.out.println("New String Array: " + Arrays.toString(new_strings));
+        Number[] new_numbers = Arrays.copyOf(numbersArray, 5, Number[].class);
+        System.out.println("New Number[] Array: " + Arrays.toString(new_numbers));
+
+        /**
+         * Lets look at the difference between method java.lang.System.arraycopy() and java.util.Arrays.copyOf
+         * Arrays.copyOf() creates a new array object when copying the element. System.arraycopy copies into an existing array.
+         * If we read the source code of Arrays.copyOf() {@link <a> her="https://developer.classpath.org/doc/java/util/Arrays-source.html"</a>},
```

Comment:
```suggestion
         * If we read the source code of {@link java.util.Arrays#copyOf(int[], int[])},
```

Fixed link to `java.util.Arrays#copyOf ` so it lead to source code.

Reply:
Thank you.